test/parsing/vass/line-numbers-after-newline-in-string.good

```diff
@@ -1 +1 @@
-line-numbers-after-newline-in-string.chpl:3: error: Variable 'x' is not initialized or has no type
+line-numbers-after-newline-in-string.chpl:3: error: split initialization not supported for globals
```

Comment:
Hm this is a tricky one. The `is not initialized or has no type` message makes a lot more sense to a beginner, whereas `split initialization not supported for globals` is for advanced users and will confuse a beginner.

I suggest keeping the `is not initialized or has no type` error and adding USR_PRINT(`split initialization is not supported for globals`).